api/app/tag.py

```diff
@@ -246,11 +253,47 @@ def make_image(payload: bytes, frame_type: Literal["simple", "secure"]) -> bytes
         image_factory=StyledPilImage,
     )
 
-    # TODO: Wrap in frame using cairosvg + pillow
+    # Get the dimensions of the image in pixels
+    img_width, img_height = img.size
+
+    # Calculate the new image dimension
+    percentage_modifier = 1.35
+    new_width = int(img_width * percentage_modifier)
+    new_height = int(img_height * percentage_modifier)
+
+    if frame_type == "secure":
+        frame_path = signed_tag_frame
+    else:
+        frame_path = simple_tag_frame
+
+    # Load SVG frame and convert to PNG
+    with open(frame_path, "rb") as svg_file:
+        svg_data = svg_file.read()
+        png_data = cairosvg.svg2png(
+            bytestring=svg_data,
+            background_color="#FFFFFF",
+            output_width=new_width,
+            output_height=new_height,
+        )
+        frame = Image.open(BytesIO(png_data))
+
+    # Create a new image with the calculated dimensions
+    new_image = Image.new("RGB", (new_width, new_height), color=(255, 255, 255))
+
+    # Paste the frame onto the new image
+    new_image.paste(frame, (0, 0))
+
+    # Calculate the position to draw the QR code within the frame
+    y_correction = img_height // 20
```

Comment:
```suggestion
    y_correction = int(img_height * 0.05)
```
multipliers are clearer than division in this kind of case